### ISSUE TITLE
Issue with the parameter select

### ISSUE THREAD

**Issue by Cobra5197** [neo4j-labs/neodash#678]
Hello, 

- **NeoDash version:** 2.3.5
- **Neo4j Database version:** 5.12.0 (Enterprise).

The issue happened with the report Parameter Select (Custom query). 

I'm using a query that returns property values that are integers (the problem happened with numbers composed of more than 4 digits):
```
MATCH (season:Season) 
RETURN season.season
```

When I use the parameter from the parameter select report in other queries, even if I force the conversion to an interger, it doesn't work. But If I change the query in the parameter select to:
```
MATCH (season:Season) 
RETURN toString(season.season)
```
The queries are working.

It looks like there is an issue on how are displayed and stored values from the custom query of the parameter select, it's not an integer but a string with a space in it. Example: 2024 became 2 024. So even If I convert 2 024 to an integer in the queries, it will never work.

Best regards,
Cobra

**Comment by JonanOribe** [comment]
Hi @Cobra5197, I will take a look to this one. Thanks for the information!!

**Comment by nielsdejong** [comment]
This appears to be an issue caused by the French locale for number formatting. (spaces as thousand-seperators)
Fix is added here, and will be release in v2.4.2.
https://github.com/neo4j-labs/neodash/pull/744
